doc/sidecar_configmap.md

```diff
@@ -69,35 +71,39 @@ data:
         command:
           - /tmp/scripts/wait-fuse.sh
 ```
-> For more sample ConfigMap files, see [examples](https://github.com/pingcap/chaos-mesh/tree/master/examples/chaosfs-configmap)
 
-Description of `chaofs-tikv.yaml`: 
+For more sample ConfigMap files, see [examples](https://github.com/pingcap/chaos-mesh/tree/master/examples/chaosfs-configmap).
+
+Description:
 
 * **name**: defines the name of the sidecar config, this name should be unique across all sidecar configs.
 * **selector**: is used to filter pods to inject sidecar.
 * **initContainers**: defines the [initContainer](https://kubernetes.io/docs/concepts/workloads/pods/init-containers/) need to be injected.
-* **container**: defines the sidecar container need to be injected. 
+* **container**: defines the sidecar container need to be injected.
 * **volumeMounts**: defines the new volumeMounts or overwrite the old volumeMounts of the each containers in target pods.
 * **volume**: defines the new volumes for the target pod or overwrite the old volumes in target pods.
-* **postStart**: called after a container is created first. If the handler fails, the containers will failed. 
+* **postStart**: called after a container is created first. If the handler fails, the containers will failed.
 Key defines for the name of deployment container. Value defines for the Commands for stating container.
 
 ### Containers
 
-#### chaosfs
+#### `chaosfs`
```

Comment:
Do we need this?

Reply:
You mean the backtick(`)? I am not sure, actually. But the following paragraphs use like that.

Reply:
This seems odd in Markdown. I think it is better to remove the backticks. What do you think?

Reply:
It's totally OK to use backticks in titles. The point is to use them consistently. 

We often use backticks to frame the following elements:

* Variable name
* Variable value
* Error message
* Configuration item name
* Field name
* The element that needs emphasis (for example, table name, column name, schema name, etc.)